helm-chart-sources/pulsar/templates/broker-deployment/broker-transactions-metadata.yaml

```diff
@@ -15,7 +15,9 @@
 #
 #
 {{- if .Values.extra.broker }}
-{{- if and .Values.broker.transactionCoordinator.enabled (or .Release.IsInstall .Values.initialize) }}
+{{- if .Values.extra.broker }}
```

Comment:
Is there an extra line here?

Reply:
I agree, it looks like we might need another `{{- end }}` since we're adding an `if`.